worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/TierWriterHelper.java

```diff
@@ -17,29 +17,24 @@
 
 package org.apache.celeborn.service.deploy.worker.storage;
 
+import org.apache.hadoop.conf.Configuration;
+
 import org.apache.celeborn.reflect.DynConstructors;
 import org.apache.celeborn.server.common.service.mpu.MultipartUploadHandler;
 
 public class TierWriterHelper {
   public static MultipartUploadHandler getS3MultipartUploadHandler(
-      String bucketName,
-      String s3AccessKey,
-      String s3SecretKey,
-      String s3EndpointRegion,
-      String key,
-      int maxRetryies) {
+      Configuration conf, String bucketName, String key, int maxRetryies) {
     return (MultipartUploadHandler)
         DynConstructors.builder()
             .impl(
                 "org.apache.celeborn.S3MultipartUploadHandler",
-                String.class,
-                String.class,
-                String.class,
+                Configuration.class,
```

Comment:
Pass the Hadoop file system instance here, and you won't need to modify the worker's pom file.